pkgs/applications/networking/browsers/chromium/common.nix

```
@@ -127,6 +128,7 @@ let
       ++ optionals gnomeSupport [ gnome.GConf libgcrypt ]
       ++ optionals cupsSupport [ libgcrypt cups ]
       ++ optional pulseSupport libpulseaudio
+      ++ optional (VAAPISupport && (versionRange "72" "73")) libva
```

Does versionRange clamp to 72-73? Don't we probably want it for all versions > 72 ?

---

yeah I guess we could replace that by a 
```suggestion
      ++ optional (VAAPISupport && (versionAtLeast version "73")) libva
```
as future versions will, whatever is the patchset required, need libva build dep.